sdk/runtime/runenv.go

```diff
@@ -30,6 +30,7 @@ const (
 	EnvTestInstanceCount  = "TEST_INSTANCE_COUNT"
 	EnvTestInstanceRole   = "TEST_INSTANCE_ROLE"
 	EnvTestInstanceParams = "TEST_INSTANCE_PARAMS"
+	EnvTestArtifacts      = "TEST_ARTIFACTS"
```

Comment:
Let's rename in accordance with #439 ;-)

Reply:
I will leave that for a future PR, because there are multiple occurrences of the word artifacts, and it appears there is no consensus on the name.